general_rules/Scenario.tex

```diff
@@ -127,9 +127,18 @@ \subsection{Objects}
 \paragraph*{Important note:} It is not allowed to modify any of the objects provided for training. Also, teams are not allowed to keep more than 5 the objects provided for training at a time nor retaining it for more than one hour.
 
 \subsubsection{Containers}
-The TC will provide at least two containers (a transport container such as a tray and a pouring container such as a bowl) which will be available for training during the setup days.
+The TC will provide at three different types of containers to be used in the tests.
 
-There are no restrictions on a container size, appearance or weight; however, it can be expected that the selected containers be lightweight, with handles, and easily manipulable by a human using both hands.
+\begin{itemize}
```

Comment:
Yeah, I didn't notice the change was in the general rules and not specific for this test and deleted my comment on this. 
The changes are fine.